examples/WebRadioDemo/WebRadioDemo.ino

```diff
@@ -0,0 +1,113 @@
+/**
+  Wiring:
+  --------------------------------
+  | VS1053  | ESP8266 |  Other   |
+  --------------------------------
+  |   SCK   |   D5    |    -     |
+  |   MISO  |   D6    |    -     |
+  |   MOSI  |   D7    |    -     |
+  |   XRST  |    -    |    -     |
+  |   CS    |   D1    |    -     |
+  |   DCS   |   D0    |    -     |
+  |   DREQ  |   D3    |    -     |
+  |   5V    |    -    |   VCC    |
+  |   GND   |    -    |   GND    |
+  --------------------------------
+
+  Dependencies:
+  -VS1053 library by baldram (https://github.com/baldram/ESP_VS1053_Library)
+  -ESP8266Wifi
+
+  Instructions:
+  -Build the hardware
```

Comment:
Maybe some link to your project? ;-) If you like of course...
So one section might be extended like this:

```
-Build the hardware
  (please find an additional description and Fritzing's schematic here:
  https://github.com/vincegellar/Simple-Radio-Node#wiring)
```